pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -1085,20 +1082,18 @@ func TestTLSClassification(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if tt.teardown != nil {
-				t.Cleanup(func() {
-					tt.teardown(t, tt.context)
-				})
-			}
-			if tt.preTracerSetup != nil {
-				tt.preTracerSetup(t, tt.context)
-			}
-			tr := setupTracer(t, cfg)
 			if tr.ebpfTracer.Type() == connection.EBPFFentry {
 				t.Skip("protocol classification not supported for fentry tracer")
 			}
-			tt.postTracerSetup(t, tt.context)
-			tt.validation(t, tt.context, tr)
+			t.Cleanup(func() { tr.removeClient(clientID) })
+			t.Cleanup(func() { tr.ebpfTracer.Pause() })
+
+			tr.removeClient(clientID)
+			initTracerState(t, tr)
+			require.NoError(t, tr.ebpfTracer.Resume(), "enable probes - before post tracer")
+			tt.postTracerSetup(t)
+			require.NoError(t, tr.ebpfTracer.Pause(), "disable probes - after post tracer")
```

Comment:
Probably it id not related to this PR, but I think it is confusing to use the function Pause, while it disable the probes

Reply:
I can agree, but unfortunately it is in ebpf-manager
you're welcome to open a PR in that repo and rename the functions to disable/enable